environment.yml

```diff
@@ -15,6 +15,8 @@ dependencies:
   - pandas
   - zeo
   - zodb
+  - openssl
+  - libexpat
```

Comment:
I am reluctant to add explicit dependencies to such standard packages, which implicitly comes with other dependencies. This may impact our conda package builds.